### PR TITLE
[Backport] Add logging command line option to ledger http service (#9581)

### PR DESCRIPTION
**Pull request by realvictorprm**
Part of #11604

CHANGELOG_BEGIN

- [Ledger HTTP Json service] Logging can now be configured via the `--log-level` cli argument. Valid values are `error`, `warn`, `info` (default), `debug`, `trace`

CHANGELOG_END

Co-authored-by: victor.mueller@digitalasset.com <mueller.vpr@gmail.com>

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger-service/http-json-oracle/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# To start a suitable oracle instance locally just use
+`docker run -d --rm --name oracle --network host -e ORACLE_PWD=hunter2 digitalasset/oracle:enterprise-19.3.0-preloaded-20210325-22-be14fb7`
```
Comment (realvictorprm) [inline ledger-service/http-json-oracle/README.md:2]: feck, this shouldn't be in there :facepalm:

**Comment by realvictorprm** [comment]
OK I think the build should succeed now, this is the first bunch of commits that backport the logging level cli option & improved logging messages for the json api. Relevant past PR's are #9581 #9592 #9618.

**Comment by stefanobaghino-da** [comment]
One of the items in the checklist above asks you to link this to the relevant issue that you are working on. If this PR closes the issues, please do so as explained in [GitHub's documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). If it doesn't, please do mention it in a way that doesn't close the ticket. This way we can easily track what has been done in order to close a PR. Thanks.

**Comment by victormueller-da** [comment]
> One of the items in the checklist above asks you to link this to the relevant issue that you are working on. If this PR closes the issues, please do so as explained in [GitHub's documentation](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue). If it doesn't, please do mention it in a way that doesn't close the ticket. This way we can easily track what has been done in order to close a PR. Thanks.

I noticed that I should have at least linked it. While going through the check list I only thought about the part "does this solve the issue completely? - no it doesnt so no need to add it" :sweat_smile: Will link it in the pr description.

**Review by akshayshirahatti-da — APPROVED** [review]
LGTM

**Pull request merged**